src/photosui.cs

```diff
@@ -580,5 +591,17 @@ interface PHPhotoLibrary_PhotosUISupport
 	{
 		[Export ("presentLimitedLibraryPickerFromViewController:")]
 		void PresentLimitedLibraryPicker (UIViewController controller);
+
+		[iOS (15,0), MacCatalyst (15,0)]
+		[Export ("presentLimitedLibraryPickerFromViewController:completionHandler:")]
+		void PresentLimitedLibraryPicker (UIViewController controller, Action<string[]> completionHandler);
```

Comment:
Looking at the docs, this looks like an Async candidate: https://developer.apple.com/documentation/photokit/phphotolibrary/3752108-presentlimitedlibrarypickerfromv

Reply:
The new swift async/await makes it easy to know, from the docs:

```
You can call this method from synchronous code using a completion handler, as shown on this page, or you can call it as an asynchronous method that has the following declaration:
```

Reply:
Ah good call, are most completionHandlers a good indicator of an async method?

Reply:
Most of the time, the new docs for swift make it easier to know. Intro will complain sometimes if it has the info.

Reply:
Updated, but forgot to run the tests, I'll do that now

Reply:
Okay they look like they are passing!